### PR TITLE
Oscal optimization

### PR DESCRIPTION
**Pull request by davidpofo**

### CODE REVIEW

**Review by gregelin — COMMENTED** [review]
Looks good!

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v0.9.1.43
+v0.9.1.43.X
```
Comment (davidpofo) [inline VERSION:1]: Needs a number to pass circleci

**Comment by gregelin** [comment]
Yes I put .X in as a placeholder. Feel free to change.



Greg Elin
CEO, GovReady PBC
p: 917-304-3488
e: gregelin@govready.com

**Review by gregelin — APPROVED** [review]
Looks great!

**Pull request merged**